packages/twenty-server/src/engine/api/graphql/graphql-query-runner/helpers/process-nested-relations-v2.helper.ts

```diff
@@ -147,8 +147,11 @@ export class ProcessNestedRelationsV2Helper {
 
     const targetObjectRepository = workspaceDataSource.getRepository(
       targetObjectMetadata.nameSingular,
-      shouldBypassPermissionChecks,
-      roleId,
+      shouldBypassPermissionChecks
```

Comment:
Like in other places where you've addressed that feedback already, I would have tried to introduce rolePermissionConfig at the level above already to avoid that kind of nested ternaries